### ISSUE TITLE
Release policies

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/simtools#526]
We have very few releases of simtools, although a lot of development is going on.

Last release 0.4.0 was in November 2022 (https://github.com/gammasim/simtools/releases).

I would suggest that we start thinking about a release policy.

- release when we feel it is time for and a major changed happened
- define milestones and try to release 2-4 times per year after those milestones are reached
- release frequently (once per month or so); could do this either by hand (allows to decide if we have a minor or major change) or automatically.

Something to discuss in a meeting.

**Comment by orelgueta** [comment]
I would vote for the first or second option (or a combination of both). I don't see a reason for us to release every month right now considering our long-term plans and user base.

**Comment by GernotMaier** [comment]
OK, let's have a meeting next week when Victor is back and discuss 0.5.0. 

We probably should start using the 'milestone' option in github, I've started one for 0.0.5 here: https://github.com/gammasim/simtools/milestone/3

**Comment by VictorBarbosaMartins** [comment]
I agree that defining the milestones and releasing after them is a good idea. Although not too important, I think that the version number does indeed deliver a subtle message on how developed the code is. Therefore, I agree with releasing more often
